src/main/java/edu/SlowestKeySolution.java

```diff
@@ -1,9 +1,78 @@
 package edu;
 
+import java.util.Arrays;
+
 public class SlowestKeySolution {
     public static final char INVALID = '-';
 
+
+//    String keysPressed = "asdagrehhg";
+//    int[] releaseTimes = new int[] { 1, 3, 4, 5, 8, 9, 10, 13, 14, 17};
+//        SlowestKeySolution.solve(releaseTimes, keysPressed);
+
     public static char solve(int[] releaseTimes, String keysPressed) {
```

Comment:
Whole this method can be refactored into smaller ones for easy understanding and better testing

Reply:
My this evening task numb. 1 :))